reference-guide/bpmn.adoc

```diff
@@ -0,0 +1,5 @@
+= BPMN Reference
+include::../partials/attributes.adoc[]
+
+// TODO
+Please see the https://add-link[full reference guide] available at ...
```

Comment:
Remove 'Please'
Keep it as 'See the https://.....'
OR 
For more information, see the https:// [This is what we follow in documentation]